EIPS/eip-5750.md

```diff
@@ -1,18 +1,18 @@
 ---
 eip: 5750
-title: Extra Data Parameter in Methods
-description: Defines an extra data parameter in methods.
+title: Extending Method Behavior with Data Field
```

Comment:
Reviewers, I really need some good suggestion for title. :) Suggestions are greatly appreciated.